pavelib/quality.py

```diff
@@ -234,7 +241,7 @@ def run_complexity():
     Uses radon to examine cyclomatic complexity.
     For additional details on radon, see http://radon.readthedocs.org/
     """
-    system_string = 'cms/ lms/ common/ openedx/'
+    system_string = join(ALL_SYSTEMS, '/ ') + '/'
```

Comment:
I'm not sure I understand the changes made in this file, can you explain? Looks like `/pavelib` is now being linted for complexity?